templateflow/conf/__init__.py

```diff
@@ -88,20 +88,24 @@ def _update_datalad():
 
 def init_layout():
     from .bids import Layout
+    from bids.layout import BIDSLayoutIndexer
 
     global TF_LAYOUT
     TF_LAYOUT = Layout(
         TF_HOME,
         validate=False,
```

Comment:
```suggestion
```

Reply:
BIDSLayout also takes in validate - AFAIK this is still required.

Reply:
It should be ignored if an indexer is passed.